gson/src/main/java/com/google/gson/internal/bind/JsonTreeWriter.java

```diff
@@ -199,6 +200,16 @@ private void put(JsonElement value) {
     return this;
   }
 
+  /**
+   * {@inheritDoc}
+   *
+   * @since 2.8.7
+   */
+  @Override public JsonWriter jsonValue(String value) throws IOException {
+    put(JsonParser.parseString(value));
```

Comment:
I don't have full context here but do we want to check for null value? 
I see other methods do 
 if (value == null) {
      return nullValue();
    }